spec/index.bs

```diff
@@ -374,13 +409,12 @@ configuration is suitable for the application.
 <dl dfn-type=dict-member dfn-for=GPURequestAdapterOptions>
     : <dfn>powerPreference</dfn>
     ::
-        Provides a hint indicating what class of [=adapter=] should be selected from the system's
-        available adapters.
-
-        If unspecified, the user agent decides which class of adapter is most suitable.
+        Provides a hint indicating what class of [=adapter=] should be preferred.
```

Comment:
IDK that having a hint helps, if we have the `GPUAdapterList` then maybe we can just return all the adapters the application can query and let them decide which one they want to use?

Reply:
As long as we always return low power adapters first, I suppose this would work. It would mean apps looking for the high performance gpu would have some fairly complex logic to find it though.

Reply:
not that bad I guess:

```js
let adapter = adapters[0];
for (const a of adapters) {
  if (a.powerProfile === 'high-performance') {
    adapter = a;
  }
}
```